### ISSUE TITLE
Slash menu style is distorted on Next.js project

### ISSUE THREAD

**Issue by i-am-chitti** [TypeCellOS/BlockNote#670]
**Describe the bug**
I used this library with a fresh Next.js @ 14.1.4 release. On loading the editor as guided here- https://www.blocknotejs.org/docs/advanced/nextjs, the slash menu style is broken - 
<img width="831" alt="image" src="https://github.com/TypeCellOS/BlockNote/assets/60139930/14bfdac0-5902-4784-ab70-ac90f213a19c">

Also, noticed that this style is broken - 
<img width="422" alt="image" src="https://github.com/TypeCellOS/BlockNote/assets/60139930/285b179e-d28a-4022-a224-bcd66b91aca2">

**To Reproduce**
1. Create a next js project - 
```
npx create-next-app@latest blocknote-next-js --typescript --eslint
```
2. ```cd blocknote-next-js```
3. Add blocknote lib - `npm install @blocknote/core @blocknote/react`
4. Follow the guide - https://www.blocknotejs.org/docs/advanced/nextjs and add the Editor
5. Start development server
6. Press `/` and you'll get the distorted styling

**Misc**
- Node version: 18.18.0
- Package manager: 9.8.1
- Browser: Google Chrome - Version 123.0.6312.59 (Official Build) (arm64)

**Note**
I'm using the exact same code as suggested in the doc. Have already imported the "react/style.css"

**Comment by quuu** [comment]
Am also hitting this issue on a fresh repo

**Comment by quuu** [comment]
The stackblitz linked in the docs also is directly repro-able for this issue

![CleanShot 2024-03-29 at 17 19 22@2x](https://github.com/TypeCellOS/BlockNote/assets/32676955/ab6a9112-37b4-4215-acb7-75176cd38a72)

**Comment by siddharthbulia** [comment]
Same issue here. It is not working in regular project as well.

**Comment by siddharthbulia** [comment]
@i-am-chitti and @quuu 

I have been to fix it by installing `@mantine/core@7.5.0` and by adding `import "@mantine/core/style.css` at the start of my App.jsx
It seems like this file should have automatically been imported but is not happening. 


```
import "@blocknote/core/fonts/inter.css";
import { BlockNoteView, useCreateBlockNote } from "@blocknote/react";
import "@blocknote/react/style.css";
import "@mantine/core/style.css"

export default function App() {
  // Creates a new editor instance.
  const editor = useCreateBlockNote();

  // Renders the editor instance using a React component.
  return <BlockNoteView editor={editor} />;
}

```

**Comment by kyen99** [comment]
In order to get this to work with latest @blocknote I had to do what @siddharthbulia suggested above, but I had to install @mantine/core@7.7.1 instead of 7.5.0 (which gave me dependency warnings and didn't work).

**Comment by DaviRJ** [comment]
Same problem here. Nextjs 14.1. @siddharthbulia fixed the Blocknote layout but broke up all styles of my app 😅 I'm using Tailwind and shadcn. 

Bellow some images to exemplify.

Blocknote without manually import:

![image](https://github.com/TypeCellOS/BlockNote/assets/28058907/89d0bcde-c75c-44fd-9b27-47b3ab8508c0)
--
![image](https://github.com/TypeCellOS/BlockNote/assets/28058907/f08b297a-3be8-4565-9a3c-4cb8d66ad5ee)

Blocknote manually importing @mantime/core/styles.css:

![image](https://github.com/TypeCellOS/BlockNote/assets/28058907/04434569-6663-41f9-ac0a-a87e4e2ef5fa)

**Comment by YousefED** [comment]
Thanks for the reports. This seems like a recent regression, we'll fix it asap!

**Comment by shahyan3** [comment]
@YousefED Thanks. Do post an update when fixed please.

**Comment by SPCODER-DEV** [comment]
it works like this so far

Adding "@mantine/core/styles.css" and also importing "@/app/globals.css" 

 "@mantine/core": "^7.7.1"

**Comment by YousefED** [comment]
Hi all! Turns out this was caused by a Mantine update. It should be fixed in the latest BlockNote release! (0.12.4)

#678
